types/nationalAvalancheCenter/schemas.ts

```diff
@@ -341,7 +342,22 @@ export const externalMediaSchema = imageMediaSchema.extend({
 });
 export type ExternalMediaItem = z.infer<typeof externalMediaSchema>;
 
-export const mediaItemSchema = z.discriminatedUnion('type', [emptyMediaSchema, nullMediaSchema, imageMediaSchema, videoMediaSchema, externalMediaSchema, photoMediaSchema]);
+export const pdfMediaSchema = z.object({
+  type: z.literal(MediaType.PDF),
+  url: z.object({
+    original: z.string().url(),
+  }),
+});
+
+export const mediaItemSchema = z.discriminatedUnion('type', [
+  emptyMediaSchema,
+  nullMediaSchema,
+  imageMediaSchema,
+  videoMediaSchema,
+  externalMediaSchema,
+  photoMediaSchema,
+  pdfMediaSchema,
```

Comment:
I feel like we should have a fallback or catchall schema here for an `unknown` media type. The client currently ignores schemas it isn't set up to display (`photo`, `video`, `external`) so it should be able to ignore new media types by default.

The problem is that I couldn't figure out the magic Zod invocation to make this work... 😬